chia/data_layer/s3_plugin_service.py

```diff
@@ -18,6 +18,9 @@
 from chia.types.blockchain_format.sized_bytes import bytes32
 
 log = logging.getLogger(__name__)
+plugin_id = "2bdf9ed6-f3bc-44f6-a69d-06602c0688d9"
```

Comment:
this seems meaningless unless its dynamic since i assume each plugin will have its own version.

Reply:
Why would separate plugins be versioned to be unique from each other?  Though I suspect I'm just totally missing your intent.

Reply:
I removed this